src/components/v5/common/ActionSidebar/partials/Motions/steps/OutcomeStep/types.ts

```diff
@@ -0,0 +1,14 @@
+import { ColonyMotion } from '~types';
+import { MotionState, MotionVote } from '~utils/colonyMotions';
+
+export interface OutcomeStepProps {
+  motionData: ColonyMotion | undefined | null;
+  motionState?: MotionState;
+}
+export interface VoteStatuses {
+  id: string;
```

Comment:
Please use key instead of id to make the code consistent with the rest of the components